backend/grants/tests/test_admin.py

```diff
@@ -29,6 +61,7 @@ def test_send_reply_emails_with_grants_from_multiple_conferences(
     grant2 = grant_factory(conference=conference2, status=Grant.Status.waiting_list)
     grant3 = grant_factory(conference=conference2, status=Grant.Status.rejected)
     request = rf.get("/")
+    _setup_visa_page_and_request(request, site)
```

Comment:
what about using a fixture in this file with autouse=True?

Reply:
I thought to use a fixture but I need the `request` object anyway... 
removed as it's not needed anymore